### ISSUE TITLE
[AnotherPingCog] AttributeError in `[p]ping` command

### ISSUE THREAD

**Issue by JeffJrShim** [Vexed01/Vex-Cogs#43]
### What cog is this bug report for?

AnotherPingCog

### What versions are you running?

Version 1.1.6

### Describe the bug

Just run [p]ping, and an error will show up. 

### If there's an error, paste it here

```python
Exception in command 'ping'
Traceback (most recent call last):
  File "C:\Users\Jefferson Jr. Shim\saki\lib\site-packages\discord\ext\commands\core.py", line 85, in wrapped
    ret = await coro(*args, **kwargs)
  File "D:\saki izumo\data\cogs\CogManager\cogs\anotherpingcog\anotherpingcog.py", line 153, in ping
    settings = self.cache
AttributeError: 'AnotherPingCog' object has no attribute 'cache'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Jefferson Jr. Shim\saki\lib\site-packages\discord\ext\commands\bot.py", line 939, in invoke
    await ctx.command.invoke(ctx)
  File "C:\Users\Jefferson Jr. Shim\saki\lib\site-packages\discord\ext\commands\core.py", line 863, in invoke
    await injected(*ctx.args, **ctx.kwargs)
  File "C:\Users\Jefferson Jr. Shim\saki\lib\site-packages\discord\ext\commands\core.py", line 94, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: 'AnotherPingCog' object has no attribute 'cache'
```

**Comment by Vexed01** [comment]
Thanks for reporting this.

This isn't something that should happen "on its own". I'll push a potential fix in a few minutes, but before you update, I would really appreciate it if you looked at the bot's logs around the time you last loaded the APC cog. There is almost certainly an error there straight after the load which I would like to see, as that would be the cause of this error.

If you can't reach the logs for when you last loaded the cog, you could also try reloading the cog and seeing if there's an error in the console (it won't come up in Discord) straight after the load.

**Comment by Vexed01** [comment]
Fixed in 70334a5609e560dcfac3392d72234bd07821e67c

**Comment by JeffJrShim** [comment]
Ty vexed. Sorry, when you replied, I was asleep 😆
